utils/release/lock-master.mjs

```diff
@@ -26,6 +26,7 @@ export const removeWriteAccessRestrictions = () =>
 async function changeBranchRestrictions(onlyBot) {
   const octokit = new Octokit({auth: process.env.GITHUB_CREDENTIALS});
   // Requires branches to be up to date before merging
+  //@ts-expect-error param is a bit messed-up, it expects {data:never}, which doesn't make any sense.
```

Comment:
Temp: Used expect-error instead of ignore so that we'll be flagged when it's been fixed by GH